specification/modsimworkbench/resource-manager/Microsoft.ModSimWorkbench/preview/2021-03-01-preview/fileRequests.json

```diff
@@ -0,0 +1,493 @@
+        "description": {
+          "description": "Specifies the reason for the action.",
+          "type": "string"
+        }
+      }
+    },
+    "RequestPropertiesFormat": {
+      "description": "File requests properties.",
+      "type": "object",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp for the resource creation."
+        },
+        "fileId": {
+          "type": "string",
+          "description": "FileId for the operation"
+        },
+        "requestUserId": {
+          "type": "string",
+          "description": "Specifies the principal ID of Azure Active Directory User who requested to download."
+        },
+        "linuxUsername": {
+          "type": "string",
+          "description": "The ModSim Workbench chamber workloads local username used when login."
+        },
+        "status": {
+          "type": "string",
+          "description": "The status of ModSim Workbench chamber data pipeline request."
+        },
+        "description": {
+          "type": "string",
+          "description": "Description for the operation"
+        },
+        "approve": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "x-ms-client-flatten": true,
+              "type": "string",
+              "description": "The globally unique identifier for the approval."
+            },
+            "description": {
+              "x-ms-client-flatten": true,
+              "type": "string",
+              "description": "Description of approval"
+            },
+            "userId": {
+              "x-ms-client-flatten": true,
+              "type": "string",
+              "description": "Specifies the principal ID of Azure Active Directory User who approved to download."
+            },
+            "timestamp": {
+              "x-ms-client-flatten": true,
+              "type": "string",
+              "format": "date-time",
+              "description": "The timestamp for the approval"
+            }
+          }
+        },
+        "download": {
+          "type": "object",
+          "properties": {
+            "location": {
+              "x-ms-client-flatten": true,
+              "type": "string",
+              "description": "Download destination location."
+            },
+            "startTime": {
+              "x-ms-client-flatten": true,
+              "type": "string",
+              "format": "date-time",
+              "description": "The timestamp for the start of download."
+            },
+            "completeTime": {
+              "x-ms-client-flatten": true,
+              "type": "string",
+              "format": "date-time",
+              "description": "The timestamp for the completion of download."
+            }
+          }
+        },
+        "internalMetadata": {
+          "type": "object",
+          "properties": {
+            "sasUriId": {
```

Comment:
>sasUriId [](http://example.com/codeflow?start=13&length=8)

SAS tokens are secrets, to better mark them as x-ms-secret everywhere

Reply:
sasUriId value is not SAS URI, it's key vault secret ID for the SAS URI: not secretssasUriId is internalMetadata and should not be in Swagger